decidim-core/app/views/layouts/decidim/_process_header.html.erb

```diff
@@ -3,17 +3,19 @@
        style="background-image:url(<%= current_participatory_process.banner_image.url %>);">
     <div class="process-header__container row">
       <div class="columns mediumlarge-9 process-header__info">
-        <h1 class="text-highlight heading2">
-          <%= translated_attribute(current_participatory_process.title) %>
-        </h1>
-      </div>
-      <div class="show-for-medium">
-        <h2 class="text-highlight heading-small">
-          <span class="process-header__hashtag">
-            <%= current_participatory_process.hashtag %>
-          </span>
-          <%= translated_attribute(current_participatory_process.subtitle) %>
-        </h2>
+        <div>
+          <h1 class="text-highlight heading2">
+            <%= translated_attribute(current_participatory_process.title) %>
+          </h1>
+        </div>
+        <div>
+          <h2 class="text-highlight heading-small">
+            <span class="process-header__hashtag">
+              <%= current_participatory_process.hashtag %>
```

Comment:
Can you change this to a Twitter link?

`<%= link_to current_participatory_process.hashtag, "https://twitter.com/hashtag/#{current_participatory_process.hashtag}" %>`